### PR TITLE
Adding Ubuntu Image by Arm

### PR DESCRIPTION
**Pull request by mattlovestech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,10 +63,10 @@ Any workflows that run on a partner image do not fall under the customer [SLA](h
 - Each partner image will have a different term of support depending on the partner, partner support links can be found below
 
 
-| Partner Name | Support Link |
-|-----------|-----------------------|
-| [ARM AVH ](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |
-
+| Partner  | Image | Support Link | 
+|-----------|-----------|-----------------------|
+| Arm Limited, Inc.| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)| [About & Support](https://github.com/actions/partner-runner-images/issues/new/choose) |
```
Comment (maxim-lobanov) [inline README.md:68]: nit: I recommend adding column `OS / Architecture`:

I will be useful when we add more partner images in future.

| OS / Architecture | Partner  | Image | Support Link | 
|-|-----------|-----------|-----------------------|
| Linux, Arm64 |Arm Limited, Inc.| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)| [About & Support](https://github.com/actions/partner-runner-images/issues/new/choose) |
| Linux, x64 |Arm Limited, Inc.| [ARM AVH](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |

File: README.md [hunk 2]
```diff
@@ -21,11 +21,15 @@ Partner images are VM images which are created by third parties which GitHub Hos
 
 | Image | YAML Label | Included Software | Rollout Progress of Latest Image Release |
 | --------------------|---------------------|--------------------|---------------------|
-| [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta
+| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) | custom in larger runners| [Ubuntu 22.04 Inventory](/images/arm-ubuntu-22-image.md) | In Beta (Private)
+| [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta (Private)
```
Comment (maxim-lobanov) [inline README.md:25]: nit: I recommend adding column `OS / Architecture`:

I will be useful when we add more partner images in future.

| OS / Architecture | Image | YAML Label | Included Software | Rollout Progress of Latest Image Release |
| - | --------------------|---------------------|--------------------|---------------------|
| Linux, ARM64 | [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) | custom in larger runners| [Ubuntu 22.04 Inventory](/images/arm-ubuntu-22-image.md) | In Beta (Private)
| Linux, x64 | [ARM-Partner-AVH](https://resources.github.com/arm-gihub-actions-beta/) | custom in larger runners| [AVH Inventory](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh_inventory.html ) | In Beta (Private)

File: README.md [hunk 3]
```diff
@@ -59,10 +63,10 @@ Any workflows that run on a partner image do not fall under the customer [SLA](h
...
 
 
-| Partner Name | Support Link |
-|-----------|-----------------------|
-| [ARM AVH ](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |
-
+| Partner  | Image | Support Link | 
+|-----------|-----------|-----------------------|
+| Arm Limited, Inc.| [Ubuntu 22.04 by Arm Limited](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/)| [About & Support](https://github.com/actions/partner-runner-images/issues/new/choose) |
+| Arm Limited, Inc.| [ARM AVH](https://resources.github.com/arm-gihub-actions-beta/)| [About & Support](https://arm-software.github.io/AVH/main/infrastructure/html/avh_gh.html) |
```
Comment (maxim-lobanov) [inline README.md:69]: may be worth to merge this table with the first one? It will simplify maintenance and improve readability.

Comment (mattlovestech) [reply README.md]: @maxim-lobanov I've merged this into the main table.

**Review by maxim-lobanov — APPROVED** [review]

**Review by blg666 — APPROVED** [review]

**Pull request merged**